jira_connector.py

```diff
@@ -331,28 +331,59 @@ def _save_state(self, state):
         return phantom.APP_SUCCESS
 
     def _set_jira_error(self, result_object, message, e):
-
-        error_text = None
-
-        try:
+        # Newer versions of Jira API may not include the error messages in
+        # e.text. Instead, these messages are included in e.response,
+        # which is a Response object from requests package.
+        if hasattr(e, "text") and e.text:
```

Comment:
I'd split error text retrieval to two separate function.

Reply:
Done